<launch>

  <arg name="initial_x" default="1.0" />
  <arg name="initial_y" default="1.0" />
  <arg name="initial_yaw" default="0.0" />

  <group>
    <push_ros_namespace namespace="cmd_vel_mux"/>
    <include file="$(find-pkg-share cmd_vel_mux)/launch/cmd_vel_mux-launch.py" />
  </group>

  <node pkg="very_simple_robot_simulator" name="kobuki_simulator" exec="kobuki_simulator.py" output="screen" >
    <remap from="/cmd_vel" to="/cmd_vel_mux/cmd_vel" />
    <remap from="/active" to="/cmd_vel_mux/active" />
    <param name="initial_x" value="$(var initial_x)" />
    <param name="initial_y" value="$(var initial_y)" />
    <param name="initial_yaw" value="$(var initial_yaw)" />
  </node>

</launch>
